appengine/genetics/js/blocks.js

```diff
@@ -97,10 +97,12 @@ Blockly.Blocks['genetics_initMouseFunctionBlock_'] =
     ],
     "inputsInline": true,
     "colour": Genetics.Blocks.GENETICS_MOUSEFUNCTIONS_HUE,
-    "tooltip": BlocklyGames.getMsg('Genetics_' + funcName + 'Tooltip')
+    "tooltip": BlocklyGames.getMsg('Genetics_' + name + 'Tooltip')
   });
 };
 
+// TODO(kozbial) Add function flag to mouseFunctions once it has be implemented.
```

Comment:
In Blockly, function comments are handled differently than inline comments. (function comments use /** */ instead of // ). The function flag is something that will be added in the future so blocks like mine could be treated like functions for comments. I *might* do it, but no guarantees.